data-access/EditionRepository.cs

```diff
@@ -269,42 +276,55 @@ public async Task<string> DeleteAllEditionDataAsync(UserInfo user, string token)
             await Task.WhenAll(
                 editors.Select(
                     x => ChangeEditionEditorRights(user, x.Email, x.MayRead, false, x.MayLock, x.IsAdmin)
-                    )
-                );
-
-            using (var transactionScope = new TransactionScope())
+                )
+            );
+
+            // Note: I had wrapped the following in a transaction, but this has the problem that it can lockup every
```

Comment:
Why would deleting rows from a table lock the entire table? Is this another MySQL/MariaDB feature?